cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -233,6 +239,12 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             );
         },
 
+        callAnalytics: function (event) {
+            event.preventDefault();
+            analytics.track(event.currentTarget.className + " clicked.");
```

Comment:
@mulby I assumed we only tracked event names that were some form of "edx.bi.user.page.action" but I'm actually seeing a lot of calls to `analytics.track()` with strings like "Completed Purchase". Which one are we supposed to use here, and does it matter?

Reply:
Some names like "Order Completed" have special meaning, see: https://segment.com/docs/spec/ecommerce/v2/

"Completed Purchase" is a bit of a special case that's probably not worth going into. Suffice to say we should use the `edx.bi.foo` convention.